### DISCUSSION TITLE
provide more details on programmatically sign-in

### DISCUSSION THREAD

**Discussion by waltcow** [logto-io/logto#3877] [Q&A]
https://docs.logto.io/docs/recipes/interact-with-management-api/
there are no example on  programmatically sign-in with interact-with-management-api, 
and the api doc is missing response details, it is so hard to play with the management-api

**Comment by gao-sun** [comment]
the doc is incorrect, we don't support programmatically sign-in yet. @simeng-li can you help update it?
